static/pages/volunteer/volunteer.html

```diff
@@ -47,7 +47,7 @@ <h6 class="md-headline volunteer-page-title-box" show-xs hide-gt-xs>
       <div class="oppia-foundation-mobile-and-small-tablet-vertical-padding"
            layout="row"
            layout-align="center center">
-        <img src="/assets/images/volunteer/volunteer_page_global_graphics.png"
+        <img src="/assets/images/volunteer/Map.png"
```

Comment:
Minor, but can we have our image filenames be all lowercase (for consistency)?

Reply:
Done!